docs/design/submarine-server/architecture.md

```diff
@@ -0,0 +1,75 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+  -->
+# Submarine Server
+
+## Motivation
+Nowadays we use the client to submit the machine learning job and the cluster config is stored at the client. It’s not secure and difficult to use. So we should hide the underlying cluster system such as YARN/K8s.
```

Comment:
It is better to change "not secure" to "insecure".

Reply:
DONE